File: Baekjoon/Class2/02.30802 웰컴 키트.cpp
#include <iostream>
#include <cmath>
using namespace std;
constexpr int T_SIZE_COUNT = 6;

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);

    int n, t = 0, p, p_rest;
    int t_counts[T_SIZE_COUNT];
    int t_bundle, p_bundle;

    cin >> n;
    for (int i = 0; i < T_SIZE_COUNT; i++)
        cin >> t_counts[i];
    cin >> t_bundle >> p_bundle;

    // 티셔츠 묶음
    for (int i = 0; i < T_SIZE_COUNT; i++)
        t += int(ceil(t_counts[i] / (double)t_bundle));
        // t += (t_counts[i] + t_bundle - 1) / t_bundle;

    // 펜 묶음과 낱개
    p = n / p_bundle;
    p_rest = n % p_bundle;

    cout << t << '\n' << p << ' ' << p_rest;
    return 0;
}

File: Baekjoon/Class2/02.30802 웰컴 키트.java
import java.io.*;
import java.util.StringTokenizer;

class Main {
    private static final int T_SIZE_COUNT = 6;
    public static void main(String[] args) throws IOException {
        int n, t = 0, p, pRest;
        int[] tCounts = new int[T_SIZE_COUNT];
        int tBundle, pBundle;
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        StringTokenizer st;

        n = Integer.parseInt(br.readLine());
        st = new StringTokenizer(br.readLine());
        for(int i = 0; i < T_SIZE_COUNT; i++)
            tCounts[i] = Integer.parseInt(st.nextToken());
        st = new StringTokenizer(br.readLine());
        tBundle = Integer.parseInt(st.nextToken());
        pBundle = Integer.parseInt(st.nextToken());

        // 티셔츠 묶음
        for(int i = 0; i < T_SIZE_COUNT; i++)
            t += (int)(Math.ceil(tCounts[i] / (double)tBundle));
        // t += (tCounts[i] + tBundle - 1) / tBundle;

        // 펜 묶음과 낱개
        p = n / pBundle;
        pRest = n % pBundle;

        sb.append(t).append('\n').append(p).append(' ').append(pRest);
        System.out.print(sb);
        br.close();
    }
}